### DISCUSSION TITLE
Updates for November 1

### DISCUSSION THREAD

**Discussion by bfhealy** [ZwickyTransientFacility/scope#147] [Announcements]
## Welcome!
This is the first of a series of weekly SCoPe updates. These updates will link to PRs and issues from the previous week and briefly discuss the intended next steps of the project. Since this is the first update, please see below for an additional primer on the project status:

## Project status
- Golden training set uploaded to Fritz (group 1458) containing 80,034 unique sources.
- Code in place to download classifications from Fritz, merge with features from Kowalski to regenerate training set.
- Moving forward on re-training DNN with removed dropout layer (#53). 
  - Making improvements, fixing bugs along the way

## Pull requests
- #140
- #143
- #142
- #146

## Issues
- #136
- #137 
- #138 
- #139 
- #141 
- #144 
- #145

## Going forward
As described above (and in #54), we are ironing out some issues with the training process and the golden dataset labels. After we resolve those issues and finish training, we will run inference on ZTF sources that meet our criteria across the whole sky, make a new training set based on a validation of the highest-confidence results, and perform another round of training.

**Comment by mcoughlin** [comment]
That's a lot of progress!
